index.js

```diff
@@ -0,0 +1,62 @@
+const buttonMenu = document.querySelector('.menu-button');
+const navmenu = document.querySelector('.popup-menu');
+const main = document.querySelector('.main-content');
+buttonMenu.addEventListener('click', () => {
+  navmenu.classList.toggle('active');
+  main.classList.toggle('active');
+  buttonMenu.classList.toggle('active');
+});
+
+// Speakers SECTION
+const speakersList = [
+  {
+    name: 'Deborah Archer',
+    image: 'images/speakers/deborah-archer.avif',
+    graduate: 'President ACLU',
+    experience: 'Deborah is the president of the American Civil Liberties Union (ACLU).',
+  },
+  {
+    name: 'Johanna Faries',
+    image: 'images/speakers/Johanna.avif',
+    graduate: 'Head of Activision Blizzard',
+    experience: 'Johanna is head of Activision Blizzards Call of Dutty franchise',
+  },
+  {
+    name: 'Andrew MacAfee',
+    image: 'images/speakers/andrew.avif',
+    graduate: 'Principal Research Scientist MII',
+    experience: 'Andrew is co-founder and co-director of MIT’s Initiative on the Digital Economy',
+  },
+  {
+    name: 'Michelle Zatlyn',
+    image: 'images/speakers/Michelle.avif',
+    graduate: 'Co-founder Cloudflare',
+    experience: 'Michelle is co-founder and COO of Cloudflare,one of the world’s largest networks',
+  },
+];
+
+// console.log(speakersList)
```

Comment:
Please remove `console.log` from your codebase. It's a good practice to remove `console.log` from the codebase and also for security reasons.

Reply:
resolved